src/components/description_list/_description_list.scss

```
@@ -170,19 +170,31 @@
 
   &.euiDescriptionList--inline {
 
+    &:not(.euiDescriptionList--compressed) {
+      .euiDescriptionList__titleBadge {
+        @include fontSize($euiFontSizeS);
+      }
+    }
+
     .euiDescriptionList__title {
       @include euiFontSizeS;
       display: inline;
-      border-radius: $euiBorderRadius;
-      font-weight: $euiFontWeightRegular;
-      background: $euiColorLightestShade;
-      border: $euiBorderThin;
-      padding:  0 $euiSizeXS;
+      padding:  0;
       margin: 0 $euiSizeXS;
 
+      &Badge {
```

_Nit picky:_ Let's spell this one out completely so it's easier to find via copy/paste

```suggestion
      .euiDescriptionList__titleBadge {
```